.eslintrc.cjs

```diff
@@ -53,7 +56,7 @@ module.exports = {
     'formatjs/enforce-default-message': 'error',
     'formatjs/enforce-id': 'error',
     'formatjs/no-complex-selectors': 'error',
-    'formatjs/no-literal-string-in-jsx': 'error',
+    'formatjs/no-literal-string-in-jsx': 'off', // TODO: [AG] re-enable this after cleanup
```

Comment:
There are a lot of these, mostly the hardcoded Tekton resource kinds so intentionally left as literals. May revisit this rule in future if it becomes annoying but temporarily disabling for now.